content/Changelog/sdk-py.mdx

```diff
@@ -7,6 +7,15 @@ import { Callout } from 'fumadocs-ui/components/callout';
 
 This page documents the release history of the [Agentuity Python SDK](https://github.com/agentuity/sdk-py).
 
+## v0.0.102
+
+<Callout type="info">
+  Released: July 16, 2025
+</Callout>
+
+**Changes:**
+- **Added:** Enable the ability to use custom email domains for email replies
+
```

Comment:
add a callout that says:

Contact us if you would like to enable custom email addresses to your organization.